docs/features/custom-license-and-keyword-searches.md

```diff
@@ -0,0 +1,211 @@
+```
+
+Without the `(?m)`, it would only match at the beginning of a file.
+
+If you wanted to allow some optional whitespace before "Permission", you could add a `\s*` after the  `(?m)` flag, as `\s*` matches zero or more space characters:
+
+```
+(?m)^\s*Permission is hereby granted, free of charge
+```
+
+Finally, if you also wanted to allow an optional comment delimiter before the license, you could do this:
+
+```
+# is for Bash, Perl, Ruby, etc
+/*, // and * are for c-like comments
+-- is for Haskell
+(?m)\s*(#|/\*|\*|//|--)?\s*Permission is hereby granted, free of charge
+```
+
+### Matching to a year
+
+To match to a year, you can use the number [character class](./custom-license-and-keyword-search-regular-expression-syntax.md#perl-character-classes-unicode-friendly)) four times. This will match a four digit year:
+
+```
+(?i)this document was last updated in \d\d\d\d
+```
+
+You could also specify the number of [repetitions](./custom-license-and-keyword-search-regular-expression-syntax.md#repetitions), by putting the number of repetitions in curly quotes (`{4}`), like this:
+
+```
+(?i)this document was last updated in \d{4}
+```
+
+These regular expressions will both match, for example, "This document was last updated in 2023".
+
+## Configuring custom-license searches for your whole organization
+
+If you want to search for the same custom licenses for every project you analyze with `fossa analyze`, you can set up custom license searches in FOSSA's admin UI.
+
+In order to do this you must have permission to edit your admin's Integration Settings. If you have this permission, you can go to the "account settings" page, click on the "Integrations" tab and then the "Custom License Scans" sub-navigation.
+
+![The settings dropdown](../assets/settings-dropdown.png)
+
+![The Custom License Scans settings page](../assets/custom-license-scans-settings-page.png)
+
+You can then add custom license searches. Once you do this, anyone in your organization who runs `fossa analyze` will run the configured custom-license searches.
+
+Any custom-license searches configured in the repositories `.fossa.yml` file will also be run.
+
+### Escape characters in custom-license searches for your whole organization
+
+The match criteria in the admin interface should be escaped with single backslashes (`\`). For example, if you wanted to match a phrase containing a four-digit year, you would use
+
+```
+(?i)this document was last updated in \d\d\d\d
+```
+
+### Turning off organization-wide custom-license searches
+
+You can ignore the organization wide custom-license searches by providing the `--ignore-org-wide-custom-license-scan-configs` flag when you run `fossa analyze`:
+
+```
+fossa analyze --ignore-org-wide-custom-license-scan-configs
+```
+
+You can also set the `ignoreOrgWideCustomLicenseScanConfigs` flag to `true` in your `.fossa.yml` file. For example:
+
+```yaml
+version: 3
+ignoreOrgWideCustomLicenseScanConfigs: true
+```
+
+
```

Comment:
I recommend adding a troubleshooting section with a link to https://regex101.com/ advising them to try out their regexes on non-sensitive data with the "Rust" setting selected. Hopefully this will help folks to make sure that their regexes are correct before concluding that we are wrong.

Reply:
Done!